### PR TITLE
Shamuna

### PR DESCRIPTION
**Pull request by interstates21**

### CODE REVIEW

File: shamuna/van.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\600x789.webp">
+        </div>
+        <div class="three">
+        <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\600x658.webp">
+        </div>
+        <div class="four">
+         <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\ac_van_gogh_slider_comp11.jpg">
+        </div>
+        <div class="five">
+          <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\800x1028.webp">
```
Comment (interstates21) [inline shamuna/van.html:38]: This path won't work for anyone else, use a relative path '../img/something.jpg'

File: shamuna/van.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    </div>
+   
+    <div class="container">
+      <div class="small-container">
+        <div class="one">
+            <h2>Art</h2>
+<p>Explore Vincent van Gogh's famous paintings and drawings, work by his artist friends and contemporaries, French and Japanese prints and more.</p>
+        
+        </div>
+        <div class="two">
```
Comment (interstates21) [inline shamuna/van.html:28]: I think we should use the same class cause the cards are similar

File: shamuna/van.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    align-items: center;
+    overflow-x: scroll;
+   
+}
+.container::-webkit-scrollbar
+{
+    width:0px;
+}
+
+.small-container{
```
Comment (interstates21) [inline shamuna/van.css:38]: We don't need 2 containers here, let's remove one of them and use container instead

File: shamuna/van.html (hunk 1 above)
Comment (shamuna) [reply shamuna/van.html:38]: k

**Pull request merged**